tests/integration/cli/projects/test/tests/test_fixture_isolation.py

```diff
@@ -27,3 +27,7 @@ def test_isolation_first(alice, bob, chain):
 def test_isolation_second(bob, chain):
     assert chain.provider.get_block("latest").number == 1
     assert bob.balance == 1_000_001 * 10**18
+
+
+def test_can_use_built_in_fixtures(chain, capsys):
```

Comment:
This is the test within in the tests that prove the bug is resolved. If you add this test on `main`, the tests in `test_test.py` fail.